### PR TITLE
Integrate mr-redis to universe as a package

### PR DESCRIPTION
**Pull request by dhilipkumars**
As instructed, raising a PR to universe branch directly.

Original PR : https://github.com/mesosphere/multiverse/pull/101

cc: @pyronicide

### CODE REVIEW

**Comment by dhilipkumars** [comment]
@pyronicide Hi hope the PR is as per your expectation.  It would be nice your comment and when you are planning to integrate it.

File: repo/packages/M/mr-redis/0/config.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "properties": {
+    "mr-redis": {
+      "description": "mr-redis specific configuration properties",
+      "properties": {
+        "dns-suffix": {
```
Comment (pyronicide) [inline repo/packages/M/mr-redis/0/config.json]: This isn't used. Please remove.

File: repo/packages/M/mr-redis/0/config.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        },
+        "cluster-size": {
+          "default": 1.0,
+          "description": "Number of the redis scheduler we should have running by mesos.",
+          "minimum": 1.0,
+          "type": "number"
+        }
+      },
+      "required": [
+        "dns-suffix",
```
Comment (pyronicide) [inline repo/packages/M/mr-redis/0/config.json]: This isn't used. Please remove.

File: repo/packages/M/mr-redis/0/marathon.json.mustache [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            "forcePullImage": true
+        },
+        "type": "DOCKER",
+        "volumes": []
+    },
+    "args": [],
+    "cpus": {{mr-redis.cpus}},
+    "mem": {{mr-redis.mem}},
+    "instances": 1,
+    "ports": [5454, 5656, 2379],
```
Comment (pyronicide) [inline repo/packages/M/mr-redis/0/marathon.json.mustache]: How does someone connect to this with a redis client?

File: repo/packages/M/mr-redis/0/marathon.json.mustache [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            "network": "HOST",
+            "forcePullImage": true
+        },
+        "type": "DOCKER",
+        "volumes": []
+    },
+    "args": [],
+    "cpus": {{mr-redis.cpus}},
+    "mem": {{mr-redis.mem}},
+    "instances": 1,
```
Comment (pyronicide) [inline repo/packages/M/mr-redis/0/marathon.json.mustache]: Shouldn't this be `mr-redis.cluster-size`? The help text suggests it is instances.

**Comment by pyronicide** [comment]
Sorry about that! Go you some new comments.

File: repo/packages/M/mr-redis/0/resource.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://github.com/mesos/mr-redis/blob/master/logo-small.png",
```
Comment (pyronicide) [inline repo/packages/M/mr-redis/0/resource.json]: These need to look like this:

`https://raw.githubusercontent.com/mesos/mr-redis/master/logo-small.png`

**Pull request merged**